libdnf/module/module_sack.cpp

```diff
@@ -145,6 +149,150 @@ BaseWeakPtr ModuleSack::get_base() const {
     return p_impl->base;
 }
 
+std::tuple<
+    std::vector<std::string>,
+    std::vector<std::string>,
+    std::vector<std::string>,
+    std::vector<std::string>,
+    rpm::ReldepList>
+ModuleSack::Impl::collect_data_for_modular_filtering() {
+    base->get_module_sack()->add_modules_without_static_context();
+    auto & allPackages = get_modules();
```

Comment:
Since we renamed `ModulePackage` to `ModuleItem`, can we not call this `allPackages`, but rather e.g. `all_module_items`? (Also, use snake_case for variables.) The same for the `packageID` variable.

Reply:
Thanks

Reply:
fixed